impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js

```diff
@@ -722,7 +791,32 @@ define([
         }
       },
 
+      {
+        priority: -5,
+        select: {
+          type: "pentaho/ccc/visual/sunburst"
+        },
+        apply: {
+          extension: {
+            valuesVisible: true
+          }
+        }
+      },
+
       // context specific rules, not defined in the global configuration
+      // Abstract
+      {
+        priority: -1,
+        select: {
+          application: ["pentaho-det"],
+          type: "pentaho/ccc/visual/abstract"
+        },
+        apply: {
+          extension: {
+            selectable: false
```

Comment:
Didn't added them because i thought you were taking this one. So, i will add them and you make the needed modifications in the view code?